src/async/async_api.c

```diff
@@ -540,8 +539,13 @@ __wt_async_flush(WT_SESSION_IMPL *session)
 	WT_ASSERT(session, async->flush_op.state == WT_ASYNCOP_FREE);
 	async->flush_op.state = WT_ASYNCOP_READY;
 	WT_RET(__wt_async_op_enqueue(session, &async->flush_op));
-	while (async->flush_state != WT_ASYNC_FLUSH_COMPLETE)
-		__wt_cond_wait(session, async->flush_cond, 100000);
+	while (async->flush_state != WT_ASYNC_FLUSH_COMPLETE) {
+		/*
+		 * No run function needed, we're only pausing for 1/10th of a
+		 * second.
+		 */
```

Comment:
OK, I've removed all of the comments. My fear was it was too easy to change for how long a wait was configured without realizing the wait would then need a run-time check function where it didn't before. However, there's now an assert in the condition variable code that should catch that happening, so we're probably safe.